### DISCUSSION TITLE
HR CAM7 simulations

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#585] [General]
We are planning a  series of CAM7 simulations varying both horizontal and vertical resolutions. Specifically, we will perform AMIP simulations with observed sea surface temperature (SST) data and SST+4K at 1° and 0.25° resolutions, with both the CAM6 and CAM7 grids (i.e, with 32, 58, 93 vertical levels). 

These simulations will be used to investigate the impact of both vertical and horizontal resolution on climate sensitivity, regional precipitation patterns, gravity waves, boundary layer aerosol-cloud interactions and hurricane activity

- Run type: AMIP type for years 2000-2010.
- Tag: we will use one of the latest tags. Specific configuration is to TBD
- Output: We are collecting the request for output

| Res       | Period    | Levels | Experiment | Physics           | Comments                         |
|-----------|-----------|--------|------------|:-----------------:|----------------------------------|
| 1 deg     | 2000-2010 | L32    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
|           |           | L58    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
|           |           | L93    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
| 0.25 deg  | 2000-2010 | L32    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
|           |           | L58    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
|           |           | L93    | SST        | cam7              |                                  |
|           |           |        | SST+4K     | cam7              |                                  |
| 0.25 deg  | 2000-2005 | L58    | SST        | cam5              | 5 or 10 years  |
|           |           |        | SST+4K     | cam5              |   |
| 0.25 deg  | 2000-2005 | L58    | SST        | prog mom off | Prognostics momentum disabled: 5 or 10 years?    |
|           |           |        | SST+4K     | prog mom off |  Do we want this too?            |

@JulioTBacmeister, @adamrher, @islasimpson, @swrneale, @PeterHjortLauritzen, @cmcclus,  @bstephens82

**Comment by cecilehannay** [comment]
# Output

- **monthly**
```
fincl1 = 'ACTNI', 'ACTNL', 'ACTREI', 'ACTREL', 'AODDUST', 'AODVIS', 'AODVISdn','BURDENBC', 'BURDENDUST', 'BURDENPOM', 'BURDENSEASALT', 
'BURDENSO4', 'BURDENSOA', 'CAPE', 'CCN3', 'CDNUMC', 'CH4', 'CLDHGH', 'CLDICE', 'CLDLIQ', 'CLDLOW', 'CLDMED', 'CLDTOT', 'CLOUD', 'CMFMC_DP', 
'CT_H2O', 'DCQ', 'DQCORE', 'DTCOND', 'DTCORE', 'DTV', 'EVAPPREC', 'EVAPSNOW', 'FCTI', 'FCTL', 'FICE', 'FLDS', 'FLNS', 'FLNSC', 'FLNT', 'FLNTC', 'FLUT', 
'FREQZM', 'FSDS', 'FSDSC', 'FSNS', 'FSNSC', 'FSNT', 'FSNTC', 'FSNTOA', 'ICEFRAC', 'LANDFRAC', 'LHFLX', 'LWCF', 'MPDICE', 'MPDLIQ', 'MPDQ', 'MPDT', 
'OCNFRAC', 'OMEGA', 'OMEGA500', 'PBLH', 'PHIS', 'PINT', 'PMID', 'PRECC', 'PRECL', 'PRECSC', 'PRECSL', 'PRECT', 'PS', 'PSL', 'PTEQ', 'PTTEND', 'Q', 
'QFLX', 'QRL', 'QRS', 'QTGW', 'RCMTEND_CLUBB', 'RELHUM', 'RVMTEND_CLUBB', 'SHFLX', 'SOLIN', 'SST', 'STEND_CLUBB', 'SWCF', 
'T', 'TAUX', 'TAUY', 'TFIX', 'TGCLDIWP', 'TGCLDLWP', 'TMQ', 'TREFHT', 'TS', 'TTGW', 'U', 'U10', 'UBOT', 'UTGWORO', 'UTGW_TOTAL', 
'V', 'VBOT', 'VTGWORO', 'VTGW_TOTAL', 'WPRTP_CLUBB', 'WPTHLP_CLUBB', 'Z3', 'ZMDQ', 'ZMDT', 'N2O', 'CO2','CFC11','CFC12', 
'AODVISdn','CCN3', 'CDNUMC', 'H2O', 'NUMICE', 'NUMLIQ','OMEGA500'
```

**daily**
```
fincl3 = 'PRECT', 'PRECC', 'FLUT', 'U850', 'U200', 'V850', 'V200', 'OMEGA500', 'TS', 'SST', 'PSL'
```

-**3 hourly**
```
fincl4 =  'PRECC','PRECL'
```

**monthly**
```
fincl5 = 'Uzm','Vzm','Wzm','THzm', 'VTHzm','WTHzm','UVzm','UWzm'
phys_grid_ctem_nfreq=-6
phys_grid_ctem_zm_nbas=120
phys_grid_ctem_za_nlat=90
```
We are collecting output. Please indicate the requested output (for instance: 
- @bstephens82: add tempest extreme
- @cmcclus: add output for southern ocean.

**Reply by megandevlan** [reply]
Request to add daily (fincl3) SHFLX, LHFLX, and PBLH to enable assessing land-atm coupling across model resolution.

**Reply by islasimpson** [reply]
Would it be feasible to output 6 hourly instantaneous U,V,T,OMEGA,Q?  Perhaps that's too much to ask but I think it would be useful to be able to do some wavenumber-frequency decomposition of waves.  If it's not possible to output he full column, perhaps we can select some pressure levels e.g., 850,500,250,100,70,50,30,10,1.  What do you think @JulioTBacmeister @brianpm ?

**Reply by islasimpson** [reply]
@megandevlan Maybe also daily SOILWATER_10CM too?

**Reply by megandevlan** [reply]
@islasimpson Definitely!! Thanks for catching that, daily SOILWATER_10CM would be necessary too :)

**Reply by swrneale** [reply]
For monthly mean maybe a few Tropopause diags. 'TROP_P','TROP_T','TROP_Z'. And a few convective diags. 'LCL','PLCL','LEL'. These are all 2D so not big.

**Reply by bstephens82** [reply]
> * @bstephens82: add tempest extreme

For TempestExtremes, @JulioTBacmeister and I would request 3-hourly instantaneous output of PSL, U850, V850, Z300 and Z500.  These are all 2d fields so hopefully that's fine.  I believe Julio will request some precip variables.

I will also request some additional variables to analyze upgradient fluxes from the new prognostic momentum flux code.  I will try to get the details for this in by tomorrow 9/10/24.

**Reply by bstephens82** [reply]
@cecilehannay the new diagnostics I would like to request require a sourcemod version of clubb_intr.F90.  Are you already going to be using a sourcemod version of that file?  Or can I just base my changes on clubb_intr.F90 from a recent tag?

**Reply by cecilehannay** [reply]
@bstephens82: I don't use a modified `clubb_intr.F90`. So please base your changes on recent tags.

**Reply by cecilehannay** [reply]
Simone requested: `AODDUSTdn`

**Comment by PeterHjortLauritzen** [comment]
If the compute hours are coming out of AMWG CSL allocation then we need to be a little careful about the high res simulations unless we don't do ne240 simulations in FY25. If we don't do ne240 simulations then we can do approximately 6 10 year simulations at 1/4 degree (and no core hours to do high res CAM7 development in FY25).

Here are the numbers from the FY25 AMWG allocation request:

![Screenshot 2024-09-09 at 2 59 35 PM](https://github.com/user-attachments/assets/7f55a52c-95b8-411c-aa66-bdb9129ed028)


![Screenshot 2024-09-09 at 2 57 42 PM](https://github.com/user-attachments/assets/578db468-7ef6-4f56-bc48-0f30bccc7825)

**Reply by adamrher** [reply]
@cecilehannay I have about 7M core hours I can offer for this work, which expires on Halloween. The project code is ```CESM0024``` and I can go ahead and add you to that.

**Comment by shieldsca** [comment]
@cecilehannay For AR tracking, it would be great to have  6 -hourly instantaneous uIVT, vIVT, TMQ, PS, U850, V850

**Comment by JulioTBacmeister** [comment]
@cecilehannay It would be great to have PRECC and PRECL at 3-hourly frequency - both instantaneous and time-averaged

Also @bstephens82 , I think all of your upgradient and other CLUBB diagnostics should be written on the native SE horizontal grid to avoid confusion introduced by remapping.  As far as the other fields I am agnostic about whether they should be regridded or not. TempestExtremes can handle either latlon or SE grids, right? 

@shieldsca do you have a preference for latlon vs SE-grid outputs?

**Reply by bstephens82** [reply]
> Also @bstephens82 , I think all of your upgradient and other CLUBB diagnostics should be written on the native SE horizontal grid to avoid confusion introduced by remapping. As far as the other fields I am agnostic about whether they should be regridded or not. TempestExtremes can handle either latlon or SE grids, right?

Yes, all of the diagnostics I'm requesting, for both tempest extremes and upgradient flux analysis, should be on the native unstructured grid.

**Reply by shieldsca** [reply]
AR tracking variables can be either, we can add regridding to the workflow during the detection process. Native grid is totally fine.

**Comment by cecilehannay** [comment]
This is started to be a bit confusing as people wants some output instantaneous versus average, interpolated versus native. 

I gathered all the info in a google doc instead. 
https://docs.google.com/document/d/1T9T8Axz1L58G0YB__QNXsx5u4rpUHEhXdFFcVQP6uJY/edit?usp=sharing

I will tag your names. 
Please check you have what you want.

**Comment by PeterHjortLauritzen** [comment]
Can you set 

```se_statefreq=576```

in the 1/4 degree runs and 

```se_statefreq=144``` 

in the 1 degree runs (I would like to monitor max/min winds and other state variables). 

Thanks

**Comment by bstephens82** [comment]
Hi @cecilehannay, I have created a SourceMod version of clubb_intr.F90 that will output some additional diagnostics for analyzing upgradient fluxes.  You can find it at 
~~~
/glade/work/stepheba/for_cecile/clubb_intr.F90
~~~
To go along with this file, could you please add the following 6-hourly instantaneous output (I included the user_nl_cam lines for the clubb_history also to make it easier for you):
~~~
<fincl stream>='um:I','vm:I','upwp:I','vpwp:I','up2_tp:I','vp2_tp:I','UPWP_UG_CLUBB:I','VPWP_UG_CLUBB:I','UPWP_UG_CT_CLUBB:I','VPWP_UG_CT_CLUBB:I','DUDZ_CLUBB:I','DVDZ_CLUBB:I','UPWP_UG_MEAN_LEV:I','UPWP_UG_MIN_LEV:I','UPWP_UG_MAX_LEV:I','VPWP_UG_MEAN_LEV:I','VPWP_UG_MIN_LEV:I','VPWP_UG_MAX_LEV:I','UPWP_UG_MEAN_HGT:I','VPWP_UG_MEAN_HGT:I'

clubb_history=.true.

clubb_vars_zt='um','vm'
clubb_vars_zm='upwp','vpwp','up2_tp','vp2_tp'
~~~

**Reply by bstephens82** [reply]
@cecilehannay I would also be fine if we just take the "u" variables from that list to save space:
~~~
<fincl stream>='um:I','upwp:I','up2_tp:I','UPWP_UG_CLUBB:I','UPWP_UG_CT_CLUBB:I','DUDZ_CLUBB:I','UPWP_UG_MEAN_LEV:I','UPWP_UG_MIN_LEV:I','UPWP_UG_MAX_LEV:I','UPWP_UG_MEAN_HGT:I'

clubb_history=.true.

clubb_vars_zt='um'
clubb_vars_zm='upwp','up2_tp'
~~~

**Reply by cecilehannay** [reply]
@bstephens82 ,
On which frequency/grid do you need: 
```history_clubb=.True.```
I think that this automatically goes on the h0 file which I currently set to `monthly` and `interpolated to lat/lon`. But I could change that.

**Reply by bstephens82** [reply]
Hi @cecilehannay, as far as I can tell from namelist_definition.xml, `clubb_history` is just a single logical that should be set to .true. in order to get diagnostics directly from CLUBB, and `clubb_vars_zm` and `clubb_vars_zt` should be set exactly as I have written them.  Regarding the 
~~~
<fincl stream>='um:I','upwp:I','up2_tp:I','UPWP_UG_CLUBB:I','UPWP_UG_CT_CLUBB:I','DUDZ_CLUBB:I','UPWP_UG_MEAN_LEV:I','UPWP_UG_MIN_LEV:I','UPWP_UG_MAX_LEV:I','UPWP_UG_MEAN_HGT:I'
~~~
I think 6-hourly instantaneous would be fine.  So you could include these fields in whichever stream contains 6-hourly instantaneous output.  From your google doc, it looks like that might be `h5`?

**Reply by cecilehannay** [reply]
@bstephens82: 
Yes, `history_clubb` is just a single logical to get the history on the h0 file. **I assume you want `monthly` (am I correct or not?) and I need to know if you want on `interpolated lat/lon` or `native`.** 
The way it is set right now, it will go on the `monthly interpolated lat/lon` because it is what I set for h0. But if you want on `native`, I could set h0 as being on native grid and do `monthly interpolated lat/lon` on another history tape. Does it make sense ?

**Reply by bstephens82** [reply]
Hi @cecilehannay, I don't believe that's how `clubb_history=.true.` works.  Setting `clubb_history=.true.` in `user_nl_cam` only **enables** the user to include output from within CLUBB into the history streams.  The reason it's needed in this case, is to put `um` and `upwp` and `up2_tp` into the "h5" stream.   As far as I know (and I checked with @adamrher about this) it should not add anything to the h0 stream unless you explicitly specify that.

**Reply by bstephens82** [reply]
And just to be clear, I don't need any output on the h0 monthly stream, everything I asked for should just be 6-hourly instantaneous.

**Reply by cecilehannay** [reply]
Sorry for the confusion. 

I went in the code and I realize we are talking about 2 different things:
- `history_clubb`: this is the behavior that I described
- `clubb_history`: this is the behavior that you described

I never realize that there was 2 namelist variables with such similar names. Good to know.

**Reply by cecilehannay** [reply]
I will set: 
```clubb_history=.true.```
as you requested.

**Reply by adamrher** [reply]
there's a history_clubb too!? omg ...

**Reply by cecilehannay** [reply]
Yes, `history_clubb = true` allows to automatically output a set of variables on the h0 file.

Similarly we have other flags

- `history_amwg` => the default amwg variables 
- `history_aerosol`  
- `history_chemistry`  
- `history_dust`  
- `history_waccm`
- `...`

I had no clue that there was a `clubb_history` flag with a very different behavior.

**Reply by bstephens82** [reply]
Yes, it's confusing.  Might even be worth renaming someday...

**Reply by cecilehannay** [reply]
I noticed that the default has been to set `ndens=1` for some tapes. 
-'1' implies output real values are 8-byte
-'2' implies output real values are 4-byte.

Do we really need  ndens=1  on some tape or was it just an oversight? 
For ne30, it didn't matter too much but not ne120, saving with ndens=2 could save quite a bit of space.

**Reply by jedwards4b** [reply]
I agree that it should be ndens=2, for which tapes is this set to 1?

**Comment by cecilehannay** [comment]
@wwieder, @ekluzek, @slevis-lmwg
We are trying to set AMIP simulation at ne120. 
```
/glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/f.e30_cam6_4_032.FMTHIST.ne120_L93.001_t1
```
**We need to set a `flanduse_timeseries`. Do you have something that we could use? If not, is it possible to create a file?**

As I am testing size of input, I tried to use the only ne120 file I found (just for testing purpose).
```
flanduse_timeseries = '/glade/campaign/cesm/cesmdata/cseg/inputdata/lnd/clm2/surfdata_esmf/ctsm5.2.0/landuse.timeseries_ne120np4_SSP5-8.5_78_CMIP6_1850-2100_c230530.nc'
```
but it doesn't seem compatible with the tag I used.

**Reply by wwieder** [reply]
I don't think you want to run with transient land use.  You can add `flanduse_timeseries = ' '` to your user_nl_clm and get it to work?

**Reply by cecilehannay** [reply]
Thanks. I tried this and it solves my issue (although now I have other issues but not in the land).

Now it is not clear why I shoudl not run transient land use. Our previous FMTHIST used transient land use. Is it incorrect?

**Reply by wwieder** [reply]
for a 10 year smoke test I don't think land use change is very important to consider, and this let's you address other non-land issues 😄 .

The other thing I'll note is that your surface dataset will also have the issues with soil texture we were discussing in the co-chair meeting this week. I think the newer file, below, should work for what you need since you're running with a stub glacier model? Is this accurate @billsacks?
`/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne120np4.pg3_hist_1850_78pfts_c240908.nc`. 

Finally, @cecilehannay raises a good point here @ekluzek  and @slevis-lmwg; we seem to have a land use time series for the mpas120 grid, but not for ne120.  I'm assuming this was intentional, but don't want to _assume_?

**Reply by billsacks** [reply]
> The other thing I'll note is that your surface dataset will also have the issues with soil texture we were discussing in the co-chair meeting this week. I think the newer file, below, should work for what you need since you're running with a stub glacier model? Is this accurate @billsacks?
> `/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne120np4.pg3_hist_1850_78pfts_c240908.nc`.

Yes, if you're running with SGLC, then I *think* you can use these new surface datasets in pre-CTSM5.3.0 code.

**Reply by adamrher** [reply]
@billsacks if that doesn't work, can I just run the ctsm5.2 mksurf tools with a two line change to resolve the soil texture issue: https://github.com/slevis-lmwg/ctsm/pull/9/files#r1747755326 ?

**Reply by billsacks** [reply]
> if that doesn't work, can I just run the ctsm5.2 mksurf tools with a two line change to resolve the soil texture issue: https://github.com/slevis-lmwg/ctsm/pull/9/files#r1747755326 ?

Yes, you can fix the issue by running with the changes in that small PR. You'll need most of the other changes in that PR, and you might as well just bring them all in - e.g., there are some declarations that you need.

**Comment by cecilehannay** [comment]
Crash in the ne120_L93.
After some issues, we were able to pass the issue with initialization. 
However, the simulation is now crashing with a non reproducible error after 16 days in one occurrence and after 17 days in another occurrence. 


There is no useful error messages in the log files (at least not useful to me).
The log files are in:
```
/glade/derecho/scratch/hannay/f.e30_cam6_4_032.FMTHIST.ne120_L93.001_t0
```

In one case the error is:
```
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/) 1:  Opened existing file 
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/) 1:  /glade/campaign/cesm/cesmdata/inputdata/atm/waccm/lb/LBC_17500116-20150116_CMIP
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/) 1:  [6_0p5degLat_c180905.nc](http://6_0p5deglat_c180905.nc/)         294
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/) 0:   sysmem size=3403.4 MB rss=1824.1 MB share=89.6 MB text=47.7 MB datastack=0.0 MB
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/) 0:   sysmem size=3419.5 MB rss=1833.2 MB share=89.4 MB text=47.7 MB datastack=0.0 MB
[dec1610.hsn.de.hpc.ucar.edu](http://dec1610.hsn.de.hpc.ucar.edu/): rank -1 died from signal 9
[dec1166.hsn.de.hpc.ucar.edu](http://dec1166.hsn.de.hpc.ucar.edu/): rank 0 died from signal 15
```
In the other, I got the error:
```
[dec0246.hsn.de.hpc.ucar.edu](http://dec0246.hsn.de.hpc.ucar.edu/) 1410:   wetdapa: negative value           12          81 -5.679344388609238E-037
[dec0246.hsn.de.hpc.ucar.edu](http://dec0246.hsn.de.hpc.ucar.edu/) 1410:  -5.679344388609238E-037  0.000000000000000E+000  0.000000000000000E+000
[dec0246.hsn.de.hpc.ucar.edu](http://dec0246.hsn.de.hpc.ucar.edu/) 1410:  -0.567934438860924       0.000000000000000E+000
[dec0246.hsn.de.hpc.ucar.edu](http://dec0246.hsn.de.hpc.ucar.edu/): rank -1 died from signal 9
[dec0225.hsn.de.hpc.ucar.edu](http://dec0225.hsn.de.hpc.ucar.edu/): rank 0 died from signal 15
```

Could this be a memory leak? 
If I remember correctly in August, there was a memory leak in PUMAS. Was it fixed? (@adamrher, @jedwards4b)  

NB: Note that this test is not doing any output yet. I was just trying to see if I could run for a one month.

**Reply by adamrher** [reply]
The two most recent runs in your directory are erroring out due to negative deposition fluxes.

I thought you were running with the ne120 finidat file:
```
/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/initdata_map/[clmi.F2000.2000-01-01.ne120pg3_mt13_simyr2000_c200728.nc](http://clmi.f2000.2000-01-01.ne120pg3_mt13_simyr2000_c200728.nc/)
```
But it looks like you're running the default finidat (which is on the f09 grid). Have you tried running with the ne120 file? Don't forget to blow out the ```init_generated_files``` directory if you do change the finidat file.

**Reply by cecilehannay** [reply]
Thanks, @adamrher. I have not changed anything with the land datasets yet except to set the
```
flanduse_timeseries = ' '  
```

I can use the ne120 `finidat`:
```
finidat = /glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/initdata_map/clmi.F2000.2000-01-01.ne120pg3_mt13_simyr2000_c200728.nc
```

Should I also set the ctsm5.3 `fsurfdat`?
```
 fsurdat = '/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne120np4.pg3_hist_1850_78pfts_c240908.nc'
 ```

One more question: If the error is due to negative deposition fluxes, is it expected that it is not reproducible? The run crashed after a very different number of timesteps.

**Reply by adamrher** [reply]
Jim and Cecile are rightly skeptical of my attribution of the crashes to the negative values in wetdepa -- those are just warnings as I can't find any endrun associated with those write statements in the code. Further, the negative values are not even associated with the surface, with bad levels consistently around level 80. Sorry for being misleading.

I checked the memory usage in these two runs via qhist, and they are about 220GB per node. Seems high @jedwards4b?

**Reply by jedwards4b** [reply]
Yes I do think it's running out of memory.   Do we have a fix yet for the clubb leak that we detected?

**Reply by adamrher** [reply]
You mean PUMAS https://github.com/ESCOMP/CAM/issues/1136 ?

Julio ran this configuration not long ago: ```/glade/work/juliob/my_cam_development/cases/cDEV_ne120pg3_F2000_L93_x01/```. He's running with NTASKS=5120, so I'd suggest we start with that.

**Reply by jedwards4b** [reply]
He's removed the run directory - so no help.   And yes that's the issue I was referring to, thanks.

**Comment by slevis-lmwg** [comment]
Not to distract, but to answer these earlier questions:

> Should I also set the ctsm5.3 `fsurfdat`?
> ```
>  fsurdat = '/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_ne120np4.pg3_hist_1850_78pfts_c240908.nc'
> ```

@cecilehannay based on my reading of previous posts, yes, use the ctsm5.3.0 surface dataset.

> we seem to have a land use time series for the mpas120 grid, but not for ne120. I'm assuming this was intentional, but don't want to assume?

@wwieder this was intentional as shown in the ["grids" spreadsheet](https://docs.google.com/spreadsheets/d/1Osq56e423CF107zhoNQ0VS7-iH_JXLF9AtCvBdXyfJ4/edit?gid=584342654#gid=584342654).

**Reply by ekluzek** [reply]
@cecilehannay and @slevis-lmwg if you point to the ctsm5.3.0 datasets you'll need other changes to get it to work.

I suggest waiting until early next week for the ctsm5.3.0 tag.

**Reply by slevis-lmwg** [reply]
Ok, yes, also because of Will's [comment above ](https://github.com/NCAR/amwg_dev/discussions/585#discussioncomment-10642759) regarding the soil texture problem in the older files.

**Reply by ekluzek** [reply]
Thanks for linking to that comment @slevis-lmwg. Looking closer, since this case is a compset using SGLC (and not DGLC or CISM) it should be fine to use this with your clm version (ctsm5.2.027).

The soil texture problem might be causing strange behavior for CTSM over some regions. So it might be worth updating the dataset.

**Reply by ekluzek** [reply]
@cecilehannay @adamrher @JulioTBacmeister @PeterHjortLauritzen (and others) is there a need for ne120 transient datasets? We could add it to the grids spreadsheet and do it say for 1979-2026 like we did the other VR grids. Let us know if you have a need for that.

Otherwise, for just running a few years using a 2000 surface dataset is going to be perfectly fine for your transient cases. If you are going to want to run that entire 1979-2026 period -- you'll want to have the landuse timeseries available. We can add it if needed, but want to force you to be careful about adding high resolution datasets if they aren't really needed because they are large.

**Reply by PeterHjortLauritzen** [reply]
Since CESM highres is a priority and we would like to tune CAM7 physics at 1/4 degree (in F-case), surface dataset 1979-2026 period would be helpful.

**Reply by wwieder** [reply]
Thanks for letting us know you need a transient surface dataset for this period, Peter.  I'd assume this is something that would be good to have in months (as opposed to weeks or days), but let us know if this inaccurate.

**Reply by cecilehannay** [reply]
We are planning to run a series of F cases at n120-L93 for the period 2000-2010. We are hoping to start these runs in the next few days. Is it short enough that we can run without transient surface dataset?

**Reply by wwieder** [reply]
I would not worry about land use for a 10 year run, especially for a proof of capability simulation.

**Reply by cecilehannay** [reply]
To clarify: This study is much more than just a demonstration of capability. 

We aim to assess the impact of both vertical and horizontal resolution on climate sensitivity, regional precipitation patterns, gravity waves, boundary layer aerosol-cloud interactions, and hurricane activity. It will be a comprehensive study that many people will contribute to

**Reply by adamrher** [reply]
I am happy to generate a transient landuse_timeseries file for a 2000-2010 folks. It's quite easy to do with the new ctsm tools.

**Reply by ekluzek** [reply]
Having @adamrher create a dataset in the short term makes sense to me. In the longer term it does sound like you still want a 1979-2026 landuse timeseries file for ne120 that we support as a standard dataset in LMWG. I just want to make sure I have that right.

The issue in CTSM is here:

https://github.com/ESCOMP/CTSM/issues/2785

**Reply by adamrher** [reply]
Yes, that's what we would prefer longer term. I've added 1979-2026 to the ne120pg3 entry in the cesm3 grids doc. I also added a note "1979-2026 landuse_timeseries file may be challenging to support due to storage limitations" ... is that a fair disclaimer?

**Comment by jedwards4b** [comment]
@cecilehannay try changing: 
NTASKS=6144

**Comment by jedwards4b** [comment]
@cecilehannay  I now have a running version in `/glade/derecho/scratch/jedwards/PFS_P6144.ne120pg3_ne120pg3_mt13.FMTHIST.derecho_intel.20240920_144335_cl7c5y` a number of changes to user_nl_clm were required.

**Comment by cecilehannay** [comment]
We are locking the discussion and starting a new one here: https://github.com/NCAR/amwg_dev/discussions/587
